src/main/webapp/scripts/script.js

```diff
@@ -134,3 +129,147 @@ function geocodeLatLng(position) {
     }
   );
 }
+
+var body_colors=new Array("#E8E8E8", "#CFCFCF", "#E6E4C1", "#DEBDBD", "#B4D1B4",
+                          "#FFEABF", "#FFD7A3", "#D6C8E0", "#B9D8ED");
+var hex_col = new Array('0','1','2','3','4','5','6','7','8','9',
+                        'A','B','C','D','E','F')
+var pause = 2000;
+var speed = 20;
+var steps = 40;
+var step = 1;
```

Comment:
https://google.github.io/styleguide/jsguide.html#features-use-const-and-let please do not use `var`, instead use `const` and `let` to define variables. See https://javascript.info/var for more detailed rationale.

Reply:
@Krechals I would say global variables are fine as soon as there is many of them and used in two functions (difficult to pass them as arguments). I will add comments to make it easier to read. @jeadorf sure

Reply:
**Global variables** are something to avoid in every language because they add **side effects** (so your code will be prone to bugs). Another problem is that in JavaScript file included in the page runs in the same scope. If you have global variables in your code, scripts included after yours that contain the same variable and function names will overwrite your variables/functions.
A solution to avoid passing those variables as arguments can be to use Objects or Classes